### DISCUSSION TITLE
Gotek in Olivetti Word Processor

### DISCUSSION THREAD

**Discussion by martin-lx** [keirf/flashfloppy#618] [Q&A] [answered]
Hi,

I am trying to replace the floppy drive of a 1992 [Olivetti PTP 820](https://typewriterdatabase.com/198x-olivetti-ptp-820.10687.typewriter) word processor with a Gotek running FlashFloppy 3.30.

The problem is that I don't have much information about the machine. I suspect that it is a clone of a Japanese or Korean word processor (it looks a lot like a Sharp FW-560) but I don't know which one. The user manual does not specify the file format used or even the floppy disk size, and I found no information about it online. The original drive was a Panasonic JU-253A083N, I think it's 720k but I am not sure.

Installing the Gotek was relatively easy except that the IDC connector has to be inserted upside down, otherwise the Olivetti will not boot. Now the Gotek powers on fine and can cycle through the disk images on my thumbdrive.

The default jumper position did not work (I got a "Floppy drive error" message on the Olivetti when trying to save a file). Moving the jumper to S1 improved the situation a bit: now the Gotek green led lights up when I try to save a file. However the Olivetti gives an error message: "No disk inserted in the drive". I tried every blank image I could find.

Can someone advise me on how to try and find the right configuration? I don't know much about floppies and drives and I am a bit lost with the settings of the config files. Should I try FF.CFG first or IMG.CFG?

**Comment by keirf** [comment, marked as answer]
Jumper at position JC (if your Gotek has that jumper position) or else `interface=ibmpc` in FF.CFG?

**Reply by martin-lx** [reply]
I don't have the JC position but it works with jumper at position S1, interface=ibmpc and a 720k image. 
Thanks a lot! 
Now I feel silly having asked for help when the solution was so simple. Hopefully it will help someone in a similar situation.

**Comment by agaricus-campestris** [comment]
Hi @martin-lx ! I met you with this installation @ Césure ;)

Just to tell you I'll try reproduce your setup with a Olivetti PTP 820 (a faded yellowish grey color version I just got this morning), I contacted you on Matrix if you wanna check your dms & share stuff about it !

٩(^ᴗ^)۶
